templates/Simple/invoice.php

```diff
@@ -88,6 +88,12 @@
 					<td><?php echo $payment_method; ?></td>
 				</tr>
 				<?php endif; ?>
+                <?php if ( $due_date = $this->get_due_date( $this->order ) ) : ?>
```

Comment:
Maybe we should use the filter below `wpo_wcpdf_after_order_data`? This way a user that has a custom Simple template can take advantage of this new feature without the need to update the template.

Reply:
I added this hook and its method in the main class since I think it's proper to have it there. Do you agree?